### PR TITLE
New package: ByteDance.TikTokEffectHouse version 2.6.2.314

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112014&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Install is not silent on all machines
image

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.installer.yaml:5]:
```suggestion
PackageVersion: 2.6.2
```

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.6.2
```

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
+PackageLocale: en-US
+Publisher: TikTok LLC
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml:7]:
```suggestion
Publisher: TikTok Inc.
```

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
+PackageLocale: en-US
+Publisher: TikTok LLC
+PackageName: TikTok Effect House
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml:8]:
```suggestion
PackageName: Effect House
```

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
+PackageLocale: en-US
+Publisher: TikTok LLC
+PackageName: TikTok Effect House
+License: PRoprietary
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ByteDance.TikTokEffectHouse
+PackageVersion: 2.6.2.314
```
Comment (Trenly) [inline manifests/b/ByteDance/TikTokEffectHouse/2.6.2.314/ByteDance.TikTokEffectHouse.yaml:5]:
```suggestion
PackageVersion: 2.6.2
```

**Comment by Trenly** [comment]
[Policy] Needs-Author-Feedback

**Comment by upintheairsheep** [comment]
As TikTok Effect House is a highly popular program, have you found any flags?

**Comment by Trenly** [comment]
> As TikTok Effect House is a highly popular program, have you found any flags?

I'm not able to go through and test every set of flags. Is there any way you can test your PRs before submitting them?

**Pull request closed without merging**